### PR TITLE
PTV-1786: Deprecate FuncInfo and related methods

### PR DESCRIPTION
**Pull request by MrCreosote**
Also some minor DRYing and cleanup

### CODE REVIEW

File: docsource/releasenotes.rst [hunk 1]
```diff
@@ -9,6 +9,7 @@ VERSION: 0.13.2 (Released TBD)
 UPDATES:
```
Comment (MrCreosote) [inline docsource/releasenotes.rst:9]: `Workspace.html` above is generated

File: lib/biokbase/workspace/client.py [hunk 2]
```diff
@@ -5035,6 +5035,7 @@ def get_all_type_info(self, mod, context=None):
```
Comment (MrCreosote) [inline lib/biokbase/workspace/client.py:5035]: generated

File: src/us/kbase/workspace/FuncInfo.java [hunk 3]
```diff
@@ -15,22 +15,8 @@
 /**
```
Comment (MrCreosote) [inline src/us/kbase/workspace/FuncInfo.java:15]: generated

File: docsource/workspacescaling.rst [hunk 4]
```diff
@@ -13,7 +13,7 @@ With type delegation, one workspace can be set as the central type handler while
 workspaces delegate type operations to that workspace. The single workspace handles all writes
 and reads to the type database; since the load on the type database is extremely low compared
 to the workspace proper, this allows scaling the high load portion of the work the Workspace
-handles. It also means that only the the type handling Workspace becomes a single point of
+handles. It also means that only the type handling Workspace becomes a single point of
```
Comment (ialarmedalien) [inline docsource/workspacescaling.rst:16]: 👍

**Review by ialarmedalien — APPROVED** [review]

**Pull request merged**